tests/php/test-class-monitor-css-transient-caching.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * Class Test_Monitor_CSS_Transient_Caching.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+use AmpProject\AmpWP\BackgroundTask\MonitorCssTransientCaching;
+use AmpProject\AmpWP\Option;
+
+/**
+ * Test MonitorCssTransientCaching.
+ */
+class Test_Monitor_CSS_Transient_Caching extends WP_UnitTestCase {
+
+	/**
+	 * Set up the tests by clearing the list of scheduled events.
+	 */
+	public function setUp() {
+		parent::setUp();
+		_set_cron_array( [] );
+	}
+
+	/**
+	 * Tear down the tests by clearing the list of scheduled events.
+	 */
+	public function tearDown() {
+		parent::tearDown();
+		_set_cron_array( [] );
+	}
+
+	/**
+	 * Test whether an event is actually scheduled when the monitor is registered.
+	 *
+	 * @covers MonitorCssTransientCaching::register()
+	 */
+	public function test_event_gets_scheduled() {
+		$this->assertFalse( wp_next_scheduled( MonitorCssTransientCaching::EVENT_NAME ) );
+
+		$monitor = new MonitorCssTransientCaching();
+		$monitor->register();
+
+		$timestamp = wp_next_scheduled( MonitorCssTransientCaching::EVENT_NAME );
+
+		$this->assertNotFalse( $timestamp );
+		$this->assertInternalType( 'int', $timestamp );
+		$this->assertGreaterThan( 0, $timestamp );
+	}
```

Comment:
This test is failing in PHP 5.6 for some reason:

```
1) Test_Monitor_CSS_Transient_Caching::test_event_gets_scheduled
AmpProject\AmpWP\Exception\FailedToRegisterBackgroundTask: Failed to schedule the event 'amp_monitor_css_transient_caching' with WordPress.
/tmp/wordpress/src/wp-content/plugins/amp/src/Exception/FailedToRegisterBackgroundTask.php:30
/tmp/wordpress/src/wp-content/plugins/amp/src/BackgroundTask/CronBasedBackgroundTask.php:49
/tmp/wordpress/src/wp-content/plugins/amp/tests/php/test-class-monitor-css-transient-caching.php:41
```

Or maybe its a WordPress≤5.0 issue:

![image](https://user-images.githubusercontent.com/134745/77122251-64141680-69fa-11ea-91dd-5f7b6e32383f.png)